tests/integration/mesos/features/sync.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Mesos Sync runner
```

Comment:
Any reason we're using bdd style tests here? Unless it provides a compelling advantage I'd vote for sticking to one style of writing tests.